src/types/indexedDB.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+
+// Imported from Flow's source code and catered to our needs
```

Comment:
Please include a comment why we are rolling our own definition, linking to a hashed URL with the original Facebook definition, and add a commit with the original definition so we can have an easy diff for any upstream changes to the definition.